csharp/sources/Valkey.Glide/Internals/ResponseConverters.cs

```diff
@@ -23,7 +23,9 @@ internal class ResponseConverters
     public static Func<object, ClusterValue<T>> MakeClusterValueHandler<R, T>(Func<R, T> converter, bool isSingleValue)
         => isSingleValue
             ? value => ClusterValue<T>.OfSingleValue(converter((R)value))
-            : value => ClusterValue<T>.OfMultiValue(((Dictionary<GlideString, object>)value).ConvertValues(converter));
+            : value => value is Dictionary<GlideString, object> dict
+                ? ClusterValue<T>.OfMultiValue(dict.ConvertValues(converter))
```

Comment:
You probably can revert this - ping has always a single value. Don't need this handler.